dpdk/vr_dpdk_knidev.c

```diff
@@ -199,6 +199,27 @@ struct rte_port_out_ops dpdk_knidev_writer_ops = {
     .f_flush = dpdk_knidev_writer_flush,
 };
 
+/* Release KNI RX queue */
+static void
+dpdk_kni_rx_queue_release(unsigned lcore_id, struct vr_interface *vif)
```

Comment:
This is the same case as above. In vr_dpdk_kni_rx_queue_init() we assign qp_release_op and then in dpdk_lcore_rxtx_release() we call dpdk_kni_rx_queue_release().